### PR TITLE
Add 64bit int support

### PR DESCRIPTION
**Pull request by seanmakesgames**
converts to a double to get it across the line and back. (ala therubyracer)
trying to do this without the patch will throw a can't convert int64 to int32 exception

### CODE REVIEW

File: ext/mini_racer_extension/mini_racer_extension.cc [hunk 1]
```diff
@@ -225,7 +225,13 @@ static Handle<Value> convert_ruby_to_v8(Isolate* isolate, VALUE value) {
 
     switch (TYPE(value)) {
     case T_FIXNUM:
-	return scope.Escape(Integer::New(isolate, NUM2INT(value)));
+        length = NUM2INT(rb_funcall(value, rb_intern("size"), 0));
+        if (length > 4)
+        {
+            return scope.Escape(Number::New(isolate, NUM2DBL(value)));
```
Comment (SamSaffron) [inline ext/mini_racer_extension/mini_racer_extension.cc]: I wonder, why not simply always give JavaScript Number::New unconditionally, that way we avoid the length check?

Comment (seanmakesgames) [inline ext/mini_racer_extension/mini_racer_extension.cc]: We could do that, but then we are marshalling in as double-precision floating point all the time, which could cause other issues. I wonder why v8 provides an "Integer" class but not an int64 class

Comment (seanmakesgames) [inline ext/mini_racer_extension/mini_racer_extension.cc]: Also, I'm not sure where the code came from initially, and why it was not done that way to begin with? Also NUM2LONG exists, but doesn't work with the Integer class which is looking for an int32

Comment (SamSaffron) [inline ext/mini_racer_extension/mini_racer_extension.cc]: I see, so we lose some precision here... ok, my call is to use NUM2LONG, then we avoid the funccall, simply check if its larger than int max value if larger then convert to double etc...

Comment (seanmakesgames) [inline ext/mini_racer_extension/mini_racer_extension.cc]: I'm on board with that. I'll update the branch

**Comment by seanmakesgames** [comment]
@SamSaffron rdy for review.

**Comment by SamSaffron** [comment]
this looks great! thanks

**Pull request merged**